source/types/zestScript/ZestStatement.ts

```diff
@@ -17,6 +17,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import {
```

Comment:
These changes could have been done in its own PR as they are independent of the notifications (I still think worth raising a new PR just adding this but fine if goes together anyway).